testkit/tests/system_api.rs

```diff
@@ -18,18 +18,19 @@ extern crate exonum_testkit;
 extern crate pretty_assertions;
 
 use exonum::{api::{private::NodeInfo,
-                   public::{ConsensusStatusInfo, HealthCheckInfo}},
+                   public::{ConnectivityInfo, ConnectivityStatus, ConsensusStatus}},
              helpers::user_agent,
              messages::PROTOCOL_MAJOR_VERSION};
 use exonum_testkit::{ApiKind, TestKitBuilder};
 
 #[test]
-fn test_healthcheck_connectivity_false() {
-    let testkit = TestKitBuilder::validator().with_validators(2).create();
+fn test_connectivity_status_false() {
```

Comment:
I'd expect more tests for this endpoint with different combinations tested (ConsensusEnabled/Connected, ConsensusDisabled/Connected, etc).

Reply:
As I can see, this tests able to check whether the endpoint return expected result and correctness of serialisation. To perform more complex test need to use sandbox, and work on it in context of another task, I think.

Reply:
Sure.